docs/howto/system_testing.md

```diff
@@ -107,6 +107,14 @@ volumes:
   mysqldata:
 ```
 
+And in the test config:
+
+```
+data_stream:
+  vars:
+  # ...
+```
+
```

Comment:
This example of test configuration was introduced under `Agent service deployer` section. Could it be moved to that section again? Is this needed to be copied here too? @agithomas